specs/prometheus/rule.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: stork
+    role: prometheus-stork-rulefiles
+  name: prometheus-stork-rules-stork.rules.yaml
+  namespace: kube-system
+spec:
+  groups:
+  - name: stork-rules
+    rules:
+    - alert: StorkNonHyperConvergedPod
+      annotations:
+        message: Application pod {{ $labels.exported_pod }} in namespace {{ $labels.exported_namespace
```

Comment:
Should those be `pod` and `namespace` instead of `exported_pod` and `exported_namespace`? Because I think that is the label you are setting in the metrics